src/JoomlaBrowser.php

```diff
@@ -145,6 +145,61 @@ public function installJoomla()
         // Wait while Joomla gets installed
         $this->debug('I wait for Joomla being installed');
         $I->waitForText('Congratulations! Joomla! is now installed.', 60, ['xpath' => '//h3']);
+    }
+
+    /**
+     * Install Joomla removing the Installation folder at the end of the execution
+     */
+    public function installJoomlaRemovingInstallationFolder()
+    {
+        $I = $this;
+
+        $I->installJoomla();
```

Comment:
So here, you are calling the normal Installation function, it will execute till the last step verify the success msg and then you would go ahead and delete the installation folder and reverify the success msg? 

Nice way!